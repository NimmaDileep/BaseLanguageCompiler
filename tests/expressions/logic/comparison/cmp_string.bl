print("a" <= "a");
print("b" <= "a");

print("a" < "a");
print("a" < "b");
print("aa" < "a");

print("a" >= "a");
print("b" >= "a");

print("aa" > "a");
print("aa" > "b");

