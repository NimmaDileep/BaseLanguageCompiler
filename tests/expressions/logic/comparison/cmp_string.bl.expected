true
false
false
true
false
true
true
true
false
